modules/cnv_controlling_vms.adoc

```diff
@@ -5,15 +5,14 @@
 [[controlvm]]
 = Controlling virtual machines
 
-Virtual machines can be started and stopped, depending on the
-current state of the virtual machine. The option to restart VMs
-is available in the Web Console only.
+You can start, stop, or restart a virtual machine, depending on its current
+state.
 
-Use the *virtctl* client utility to change the state of the virtual
+Use the `virtctl` client utility to change the state of the virtual
```

Comment:
In 3.x, system packages get **s, so this was right before.

Reply:
Haha, whoops. I will change it right back.